rapid/client/executor.py

```diff
@@ -92,15 +94,39 @@ def verify_work_request(self):
             raise Exception("No command was set.")
 
     def _create_logger(self):
-        logger = logging.getLogger("rapid")
+        log_file_name = self._get_log_file_name()
+        if not log_file_name:
+            logger = logging.getLogger('rapid')
```

Comment:
my naivete makes me ask: why not use rapid_logger from above (or why not avoid defining rapid_logger above)?

Reply:
Great point! I will fix this.